notes/05_TUTORIAL_DISTR_WORKLOADS.md

```diff
@@ -1,79 +1,59 @@
 
 ```sh
-# if you are already logged in
-oc whoami -t
+oc adm policy add-role-to-group edit system:serviceaccounts:sandbox -n sandbox
 ```
 
-```python
-# Create authentication object for user permissions
-# IF unused, SDK will automatically check for default kubeconfig, then in-cluster config
-# KubeConfigFileAuthentication can also be used to specify kubeconfig path manually
-auth = TokenAuthentication(
-    token = "XXXXX",  # replace with <YOUR_TOKEN>
-    server = "XXXXX", # replace with <YOUR_API_URL>
-    skip_tls=False    # change to True to bypass certificate
-)
-auth.login()
-```
+### Create shared storage
 
-(Recommended) Change TLS trust certificate, this will always work and prevent unnecessary hops.
+[NOTE]: This is only required if you are going to run the third notebook `2_basic_interactive.ipynb` (see below).
 
-you should use the internal K8s service as the server value
-`server = "https://kubernetes.default.svc.cluster.local:443"`
+Create a project and set env vars
 
-Shorter and easier to remember
 ```sh
-# TLS verify with https service
-server = "https://kubernetes.default",
-skip_tls=False
+oc new-project minio
+MINIO_ROOT_USER=rootuser
+MINIO_ROOT_PASSWORD=rootuser123
+```
 
-# Skip TLS verify with http service
-server = "http://kubernetes.default",
-skip_tls=True
-``````
+Install MinIO helm chart
 
-You may need to create a local-queue in your project - see the CHECKLIST_PROCEDURE "Create a local queue that points to your cluster queue"
+```sh
+helm repo add minio https://charts.min.io/
+```
 
-![NOTE]
-It may also be helpful to ignore the warnings Jupyter displays
+Deploy MinIO storage in its own namespace with a bucket for distributed workloads
 
-```python
-import warnings
-warnings.filterwarnings('ignore')
+```sh
+helm install minio --namespace minio --set replicas=1 --set persistence.enabled=false --set mode=standalone --set rootUser=$MINIO_ROOT_USER,rootPassword=$MINIO_ROOT_PASSWORD --set 'buckets[0].name=distributed-demo,buckets[0].policy=none,buckets[0].purge=false' minio/minio
 ```
 
-![NOTE]
-
-`2_basic_interactive.ipynb` will require you to upgrade the `codeflare-sdk` to the latest to avoid errors. Append a cell at the top with the following:
+### Launch notebooks
 
-```ssh
-!pip install -U pip -q
-!pip install -U codeflare-sdk -q
-```
+1. Access the RHOAI Dashboard
+1. Access the `sandbox` project
+1. Create a workebench using the `Standard Data Science` notebook and set the following environment variables:
+    1. `ACCESS_KEY` = `rootuser`
+    1. `SECRET_KEY` = `rootuser123`
+    1. `ENDPOINT_URL` = `http://minio.minio:9000`
+1. In the JupyterLab interface, click Git > Clone a Repository
+1. In the "Clone a repo" dialog, enter `https://github.com/redhat-na-ssa/codeflare-sdk`
+1. In the JupyterLab interface, in the left navigation pane, double-click codeflare-sdk.
+1. Double-click demo-notebooks.
+1. Double-click guided-demos.
+1. Execute the notebooks in order
+1. `0_basic_ray.ipynb`
```

Comment:
Indent the notebook list under
- Execute the notebooks in order:
  - 0
  - 1
  - 2